### PR TITLE
show peers in raft stat

### PR DESCRIPTION
**Pull request by liwenhui-soul**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Review by critical27 — APPROVED** [review]

File: src/interface/raftex.thrift [hunk 1]
```diff
@@ -133,6 +133,7 @@ struct GetStateResponse {
     6: LogID            last_log_id;
     7: TermID           last_log_term;
     8: Status           status;
+    9: list<binary>     peers;
```
Comment (darionyaphet) [inline src/interface/raftex.thrift:136]: `peers` have been used?

Comment (liwenhui-soul) [reply src/interface/raftex.thrift:136]: what used mean?

File: src/kvstore/raftex/RaftPart.cpp [hunk 2]
```diff
@@ -1067,6 +1067,15 @@ void RaftPart::getState(cpp2::GetStateResponse& resp) {
   resp.last_log_id_ref() = lastLogId_;
   resp.last_log_term_ref() = lastLogTerm_;
   resp.status_ref() = status_;
+  std::vector<std::string> peers;
```
Comment (pengweisong) [inline src/kvstore/raftex/RaftPart.cpp:1070]: Is the peers containing itself?

File: src/interface/raftex.thrift (hunk 1 above)
Comment (darionyaphet) [reply src/interface/raftex.thrift:136]: where is the use of it

Comment (liwenhui-soul) [reply src/interface/raftex.thrift:136]: monitor tools

Comment (pengweisong) [reply src/interface/raftex.thrift:136]: Maybe you could add some explanation in the future pr

Comment (kikimo) [reply src/interface/raftex.thrift:136]: > 

It's very important for us to retrieve raft peers info when trouble shooting some issue.
BTW, think we should also including this info in storage monitring metrics.

**Review by kikimo — APPROVED** [review]
LGTM

**Pull request merged**